modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/rolesUtils.ts

```diff
@@ -9,14 +9,16 @@ import {
 } from '@liferay/object-js-components-web';
 import {createResourceURL, fetch} from 'frontend-js-web';
 
-interface Role {
+import {Role} from './UserNotificationSettings';
+
+interface EmailNotificationRole {
```

Comment:
Hey @rijema, Instead of creating this interface here, can you create one in `types.d.ts` called `LabelNameObject`? This way we are creating an interface that can be reusable in the future, the `types.d.ts` is seen in every web-notification, so you can just use the interfaces there directly without needing to import.